### ISSUE TITLE
Unable to load in

### ISSUE THREAD

**Issue by dodim5** [rhapidfyre/Cops-n-Robbers#20]
![Screenshot 2021-04-14 200117](https://user-images.githubusercontent.com/69424507/114764731-306e0200-9d5c-11eb-8537-ed10c1800de3.png)

**Comment by rhapidfyre** [comment]
Looks like a database error, which could be likely given how out of date it is. Feel free to create a fix and pull request, and I'll push it to the main branch.

**Comment by dodim5** [comment]
Yeah I dont code.. I wanted to try the mode to see if I would be able to set it up as intended and then get some devs on it to uupdate and keep it maintained
